yoti-sdk-api/src/main/java/com/yoti/api/client/YotiClient.java

```diff
@@ -141,6 +143,12 @@ public ShareSession createShareSession(ShareSessionRequest request) throws Ident
         return identityService.createShareSession(appId, keyPair, request);
     }
 
+    public ShareSessionQrCode createShareSessionQrCode(String sessionId, ShareSessionQrRequest request)
+            throws IdentityException {
+        LOG.debug("Requesting a Share Session QR code creation...");
```

Comment:
Referring back to my previous comment. A log message without dynamic params that is bound to executed multiple times per minute/second is just going to become "spam" in logs.

At least include the `sessionId` in the message.